packages/core/src/lib/hooks/useThrelte.ts

```diff
@@ -2,5 +2,11 @@ import { getContext } from 'svelte'
 import type { ThrelteContext } from '../lib/contexts'
 
 export const useThrelte = (): ThrelteContext => {
-  return getContext<ThrelteContext>('threlte')
+  const context = getContext<ThrelteContext>('threlte')
+
+  if (context.renderer === undefined) {
```

Comment:
Actually we can test for the `context` instead of the `context.renderer`, looks cleaner.

Reply:
With an error message like this: "No Threlte context found, are you using this hook inside of <Canvas>?"

Reply:
Ah yes! Totally forgot that Svelte contexts will become undefined outside of their boundaries, will update